com.ibm.wala.core/src/com/ibm/wala/cfg/exc/inter/InterprocNullPointerAnalysis.java

```diff
@@ -11,11 +11,8 @@
 
 package com.ibm.wala.cfg.exc.inter;
 
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Map;
+import java.util.*;
```

Comment:
Not a fan of star imports 😄 If / when I get Error Prone working I'll make this fail at build time

Reply:
I agree. My IDE was trying to be helpful behind my back. I have reconfigured my IDE and will correct this in a forthcoming group of updates for this pull request.